packages/core/src/analysis/SubstanceAnalysis.ts

```diff
@@ -253,23 +267,24 @@ export const identicalTypeDecls = (
   env: Env
 ): im.Map<string, Identifier<A>[]> => {
   // pulls just the variable names from the statement's parameters
-  const idSet = ids.map((i) => i.value);
+  const idStrs = ids.map((i) => i.value);
   let options: im.Map<string, Identifier<A>[]> = im.Map();
+  log.debug(`Matching ${ids.map((i) => i.value)}`);
 
-  idSet.forEach((i) => {
-    const obj = env.vars.get(i);
-    const typeStr = obj && obj.name.value;
+  idStrs.forEach((i) => {
+    const typeStr = env.vars.get(i)?.name.value;
```

Comment:
What's the motivation behind this change?

Reply:
Mainly to clarify that this is not really a set, but a list to match with existing type constructors.